tanglegram/tangle.py

```diff
@@ -754,62 +651,28 @@ def leaf_order(link, labels=None, as_dict=True):
             return leafs_ix
 
 
-def entanglement(lindex1, lindex2, L=1.5):
-    """Calculage average displacement of leafs in dendogram 1 and 2.
-
-    Entanglement is a measure between 1 (full entanglement) and 0 (no
-    entanglement). Ignores leafs that aren't present in both dendrograms.
-
-    Parameters
-    ----------
-    lindex1,lindex2 :       dict
-                            Dictionaries mapping the labels of two dendrograms
-                            to their indices.
-    L :                     any positive number
-                            Distance norm to use for measuring the distance
-                            between the two trees. Can be any positive number,
-                            often one will want to use 0, 1, 1.5 or 2:
-                            ``sum(abs(x-y)^L)``.
-
+def entanglement(link1, link2):
```

Comment:
Seems like `L` is still accepted (and other functions use it as parameter) but `entanglement` now ignores it and just does squared distance?

Reply:
This is my mistake. L should be chosen to be 0, 1, 1.5 or 2. Because I always use L = 2 so I make it unchanged. I am fixing it.